aten/src/ATen/core/op_registration/hacky_wrapper_for_legacy_signatures.h

```diff
@@ -83,13 +84,13 @@ struct with_scattered_tensor_options<UnderlyingFuncPtr, std::enable_if_t<has_ten
     using parameters_after_tensoroptions =
         guts::typelist::drop_t<gathered_parameter_types, tensoroptions_arg_index + 1>;
 
-    using wrapper = with_scattered_tensor_options_<UnderlyingFuncPtr, parameters_before_tensoroptions, parameters_after_tensoroptions>;
+    using wrapper = with_scattered_tensor_options__<UnderlyingFuncPtr, parameters_before_tensoroptions, parameters_after_tensoroptions>;
```

Comment:
double underscore?!

Reply:
(At this point it's probably worth saying why there's two levels of indirection here.)

Reply:
+1

Reply:
I renamed it to `impl_`